consensus_encoding/src/encode/encoders.rs

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Collection of "standard encoders".
+//!
+//! These encoders should not be used directly. Instead, when implementing the
+//! [`super::Encodable`] trait on a type, you should define a newtype around one
+//! or more of these encoders, and pass through the [`Encoder`] implementation
+//! to your newtype. This avoids leading encoding implementation details to the
+//! users of your type.
+//! 
```

Comment:
```suggestion
```suggestion
//! users of your type. For doing so we provide the [`encoder_newtype!`].
//!
//! [`encoder_newtype!`]: crate::encoder_newtype 
```